test/aries-js-worker/fixtures/.env

```diff
@@ -37,11 +37,11 @@ E2E_USER_API_PORT=10082
 E2E_USER_DB_PATH=/tmp/db/aries
 
 # HTTP Resolver Configurations
-HTTP_DID_RESOLVER=sidetree@http://sidetree-jsmock:48326/document
-HTTP_LOCAL_RESOLVER_URL=http://localhost:48326/document
-HTTP_LOCAL_DID_RESOLVER=sidetree@http://localhost:48326/document
+HTTP_DID_RESOLVER=sidetree@http://sidetree-jsmock:48326/sidetree/0.0.1/identifiers
+HTTP_LOCAL_RESOLVER_URL=http://localhost:48326/sidetree/0.0.1/identifiers
+HTTP_LOCAL_DID_RESOLVER=sidetree@http://localhost:48326/sidetree/0.0.1/identifiers
 
 # sidetree mock configurations
 COMPOSE_DIR=.
-SIDETREE_MOCK_FIXTURE_IMAGE=docker.pkg.github.com/trustbloc/sidetree-mock/sidetree-mock
-SIDETREE_MOCK_FIXTURE_IMAGE_TAG=0.1.2
+SIDETREE_MOCK_FIXTURE_IMAGE=docker.pkg.github.com/trustbloc-cicd/snapshot/sidetree-mock
```

Comment:
why not release version?

Reply:
we need this fix https://github.com/trustbloc/sidetree-core-go/commit/2f2b686a83c387731f70b86ef41b04663477b1f7 after release